.github/workflows/build-lint-ansible-roles.yaml

```diff
@@ -0,0 +1,23 @@
+name: Gating (AR / RHEL)
+on:
+  push:
+    branches: [ '*' ]
+  pull_request:
+    branches: [ master ]
+jobs:
+  validate-ubuntu-ar:
+    name: Build, Lint Ansible Roles on Ubuntu Latest
+    runs-on: ubuntu-latest
```

Comment:
It was previously running on `ubuntu-20.04`, could that make a difference?

Reply:
Latest is 20.04 ATM

Reply:
I believe that the problem is in the content (I've re-based the branch today)

Reply:
Oh, :-P